docs/tutorial/index.md

```diff
@@ -9,65 +9,67 @@ Gatsby Seite basiered auf einer Starter Vorlage zu leiten. Während wir diesen
 
-> Das gesamte Tutorial ist dazu gedacht so zugänglich wie nur möglich zu sein,
-für Personen, die (noch!) keine Webentwicklungserfahrung haben - es ist nicht
-notwendig ein Experte zu sein. Wenn du lieber direkt zum Code springen möchtest,
-dann kannst du dieses Schritt-für-Schritt Tutorial überspringen und dir die
-[Schnell Start](/docs/quick-start) Seite anschauen.
+> Das gesamte Tutorial ist für für Personen gedacht, die (noch!) keine
+> Webentwicklungserfahrung haben - es ist nicht notwendig ein Experte zu sein.
+> Wenn du lieber direkt zum Code springen möchtest, dann kannst du dieses
+> Tutorial überspringen und dir die [Schnell Start](/docs/quick-start) Seite
+> anschauen.
 
 ## Gatsby Grundlagen
 
 0. [Setz deine eigene Entwicklungsumgebung auf](/tutorial/part-zero/): Wir
-   führen dich in die Kerntechnologien, die Gatsby betreiben, ein und leiten
-   dich durch das Aufsetzen deiner Entwicklungsumgebung.
-1. [Lerne die Gatsby Bausteine kennen](/tutorial/part-one/): Der Beginn neuer
-   Projekte, das Entwickeln und Veröffentlichen von Webseiten.
+   setzen deine Entwicklungsumgebung auf und erläutern die Kerntechnologien von
+   Gatsby.
+1. [Lerne die Gatsby Bausteine kennen](/tutorial/part-one/): Wir erklären,
+   wie du neue Gatsby Projekte startest, sie entwickelst und als Seite
+   veröffentlichst.
 2. [Einführung in die Nutzung von CSS in Gatsby](/tutorial/part-two/): Entdecke
-   Bibliotheken wie Typography.js und CSS Module.
+   Bibliotheken wie Typography.js und CSS Modules.
 3. [Bauen von verschachtelten Layouts in Gatsby](/tutorial/part-three/):
    Layouts sind Sektionen deiner Seite, die über verschiedene Seiten wieder
    benutzt werden, wie Kopf- und Fußzeilen.
 
 ## Fortgeschrittene Tutorials
 
-In diesen fortgeschrittenen Tutorials, wirst du lernen wie du Daten von nahezu
-überall mit Hilfe von GraphQL in deine Gatsby Seite integrieren kannst.
+In diesen fortgeschrittenen Tutorials lernst du, wie Daten von nahezu
+überall mit Hilfe von GraphQL in eine Gatsby Seite integriert werden können.
 
 4. [Abfragen von Daten in einem Blog](/tutorial/part-four/): Erstellung eines
-   Blogs und Nutzung einer GraphQL Anfrage um deinen Webseiten Titel in die
-   Kopfzeile des Blogs zu ziehen.
+   Blogs und Nutzung einer GraphQL Abfrage um deinen Webseiten Titel in der
+   Kopfzeile deines Blogs darzustellen.
 5. [Datenquellen Plugins und Darstellung von angefragten Daten](/tutorial/part-five/):
-   Nutzen eines Datenquellen Plugins um Markdown Blog Beiträge in deine Seite
-   zu ziehen und Erstellung einer Index Seite mit der Liste aller Blog Beiträge.
-6. [Umformende Plugins](/tutorials/part-six/): Nutzung eines umformenden
-   Plugins um Markdown Blog Beiträge in eine Form umzuwandeln, die der Blog
-   darstellen kann.
-7. [Programatisches Erstellen von Seiten aus Daten](/tutorial/part-seven/):
+   Hier lernst du wie du ein Quellen Plugin nutzen kannst, um Markdown
```

Comment:
i would stick with `Datenquellen`, on other places there are still `Datenquellen`

but i think the best would be to stick with "Source Plugins" and not translate it. Because then the user can better search for it in the english docs

Reply:
@muescha Unfortunately this is an ongoing discussion. So far "Source Plugins" was translated into "Quellen Plugin", but a few persons are unhappy with this, including me, because it takes away the possibility of searching for a particular problem.

So I agree with you, but I think this should be unified across all the documentation.

Reply:
I believe we should just go along. We can fix such minor issues later on.